test/c_api/value_test.cpp

```diff
@@ -642,6 +642,93 @@ TEST_F(CApiValueTest, getMapValue) {
     kuzu_flat_tuple_destroy(&flatTuple);
 }
 
+TEST_F(CApiValueTest, getDecimal) {
```

Comment:
After changing to C string as required, please re-visit this test to make sure adding all necessary calls to destructors.

Reply:
You did not add destructors as I requested. 
> After changing to C string as required, please re-visit this test to make sure adding all necessary calls to destructors.

Reply:
Also, the test's name should be changed to `getDecimalAsString` to be consistent with the function you are testing

Reply:
sorry, missed a destructor while double-checking